### PR TITLE
CHANGE(repository): Managment of openstack train

### PR DESCRIPTION
**Pull request by cdelgehier**
##### SUMMARY

- Drop of xenial's tasks
- Managment of OpenStack `train`

 ##### IMPACT
N/A

 ##### ADDITIONAL INFORMATION

### CODE REVIEW

File: tasks/Ubuntu.yml [hunk 1]
```diff
@@ -78,23 +67,23 @@
...
+    id: 5EDB1B62EC4926EA
+    keyserver: keyserver.ubuntu.com
+  when:
+    - openio_repository_manage_openstack_repository
+    #- openio_repository_openstack_release == 'train'
+
 - name: "Configure OpenStack {{ openio_repository_openstack_release }} release repository"
   apt_repository:
     repo: "deb http://ubuntu-cloud.archive.canonical.com/ubuntu {{ ansible_lsb.codename }}-updates/{{ openio_repository_openstack_release }} main"
+    update_cache: false
```
Comment (vincent-legoll) [inline tasks/Ubuntu.yml:81]: why is that needed now ? Is this a bugfix ?

Comment (cdelgehier) [reply tasks/Ubuntu.yml:81]: Yes. 
The task `name: "Update APT cache"` is here for that.
